twine/wheel.py

```diff
@@ -35,27 +27,22 @@
 )
 
 
+def find_candidate_metadata_files(names: List[str]) -> List[List[str]]:
+    """Filter files that may be METADATA files."""
+    tuples = [x.split("/") for x in names if "METADATA" in x]
+    return [x[1] for x in sorted((len(x), x) for x in tuples)]
```

Comment:
nitpick: making these stylistic changes should be reserved for a separate PR.

Reply:
Dropped this change.